R/build-home-authors.R

```diff
@@ -2,6 +2,13 @@ data_authors <- function(pkg = ".", roles = default_roles()) {
   pkg <- as_pkgdown(pkg)
   author_info <- pkg$meta$authors %||% list()
 
+  inst_path <- path(pkg$src_path, "inst", "AUTHORS")
+  if (length(inst_path) == 1 && file_exists(inst_path)) {
+    inst <- read_lines(inst_path)
```

Comment:
does it need to be Markdownified? probably not?

Reply:
I don't think we can assume this file is markdown, so the best we can do is put in a `<pre>` tag.